RELEASE/scripts/autoscend/paths/small.ash

```diff
@@ -42,6 +42,13 @@ void small_initializeSettings()
 			set_property("auto_disregardInstantKarma", "false");
 		}
 	}
+	else
+	{
+		if (auto_have_familiar($familiar[Cookbookbat]))
```

Comment:
worth it to also check we can pull any of the legendary food? Folks can forget to prep them

Reply:
I just did that and it pulled other stuff like roasted vegetable of jarlsberg. I guess it's probably not a bad idea as they might be able to run the CBB enough to make something bigger worth eating like plain calzone before it runs out of adventures.

Reply:
And done.